src/main/java/Duke.java

```diff
@@ -1,10 +1,194 @@
+                break;
+
+            case COMPLETE_COMMAND:
+                // Number of task will be after done. Hence index is 1.
+                int index = Integer.parseInt(currentUserInput.split(" ")[1]);
+                taskManager.completeTask(index);
+                Task tempTask = taskManager.tasks[index - 1];
+                outputMessages = new String[]{
+                        "Nice! I've marked this task as done:",
+                        "\t" + tempTask.toString()};
+                printMessage(outputMessages);
+                break;
+
+            case TODO_COMMAND:
+                description = currentUserInput.substring(descriptionIndex);
+                taskManager.addTask(description);
+
+                outputMessages = generateAddTaskMessages(taskManager);
+                printMessage(outputMessages);
+                break;
+
+            case DEADLINE_COMMAND:
+                timingIndex = currentUserInput.lastIndexOf(DEADLINE_PREFIX);
+
+                description = currentUserInput.substring(descriptionIndex, timingIndex);
+                // Remove "/by " by adding 4 to the starting index
+                timing = currentUserInput.substring(timingIndex + 4);
+                taskManager.addDeadline(description, timing);
+
+                outputMessages = generateAddTaskMessages(taskManager);
+                printMessage(outputMessages);
+                break;
+
+            case EVENT_COMMAND:
+                timingIndex = currentUserInput.lastIndexOf(EVENT_PREFIX);
+
+                description = currentUserInput.substring(descriptionIndex, timingIndex);
+                // Remove "/at " by adding 3 to the starting index
+                timing = currentUserInput.substring(timingIndex + 4);
+                taskManager.addEvent(description, timing);
+
+                outputMessages = generateAddTaskMessages(taskManager);
+                printMessage(outputMessages);
+                break;
+
+            default:
+                printMessage("Unknown Command! Try again!");
+            }
+        }
+
+        printExitMessage();
+
+    }
+
+    /**
+     * Generates a standard message to print when a new task is added. Retrieves the last task and prints it's
+     * information.
+     *
+     * @param taskManager TaskManager Object.
+     * @return Returns a standard String Array to be printed.
+     */
+    public static String[] generateAddTaskMessages(TaskManager taskManager) {
+        Task latestTask = taskManager.getLatestTask();
+        return new String[]{
+                "Got it. I've added this task:",
+                "\t" + latestTask.toString(),
+                String.format("Now you have %d tasks in the list.", taskManager.getNumberOfTasks())
+        };
+    }
+
+    /**
+     * Prints a line to isolate Duke's replies.
+     */
+    public static void printDivider() {
+        String s = "------------------------------------------------";
```

Comment:
You may consider renaming the 's' string variable into something more descriptive such as "separator" or "divider".